config/config.exs

```diff
@@ -1,3 +1,3 @@
-use Mix.Config
+import Config
```

Comment:
See https://github.com/NFIBrokerage/spear/pull/77, this will fail to compile on older elixir versions